src/main/java/gui/RootPane.java

```diff
@@ -45,7 +45,7 @@ public void start(Stage primaryStage) throws Exception {
         // created. Also has some useful settings.
         //TODO: change this line of code.
```

Comment:
Good question, previous code review someone insisted on it being there. I've removed it.